packages/datagateway-common/src/table/columnFilters/dateColumnFilter.component.test.tsx

```diff
@@ -382,15 +418,15 @@ describe('Date filter component', () => {
         },
       };
 
-      const wrapper = mount(<DateColumnFilter {...baseProps} />);
+      render(<DateColumnFilter {...baseProps} />);
 
-      expect(wrapper.find('p.Mui-error')).toHaveLength(2);
-      expect(wrapper.find('p.Mui-error').first().text()).toEqual(
-        'Invalid date range'
-      );
+      const errorMessages = await screen.findAllByText('Invalid date range');
+      for (const element of errorMessages) {
+        expect(element).toBeInTheDocument();
+      }
     });
 
-    it('useTextFilter hook returns a function which can generate a working text filter', () => {
+    it('useTextFilter hook returns a function which can generate a working text filter', async () => {
```

Comment:
Minor comment - but this test has the wrong name - presumably because it was copied from the text filter tests and was forgotten